src/main.cpp

```diff
@@ -332,7 +332,7 @@ int main(int argc, const char* argv[]) {
         "LLVM explicit vectorisation width ({})"_format(llvm_vec_width))->ignore_case();
     llvm_opt->add_option("--veclib",
                          vector_library,
-                         "Vector library for maths functions ({})"_format(vector_library))->check(CLI::IsMember({"Accelerate", "libmvec", "MASSV", "SVML", "none"}));
+                         "Vector library for maths functions ({})"_format(vector_library))->check(CLI::IsMember({"Accelerate", "libsystem_m", "libmvec", "MASSV", "SLEEF", "SVML", "none"}));
```

Comment:
I also missed `libsystem_m` when initially this option was added. 
I haven't used `libsystem_m` in the past. Have you tried this locally? I will give a try!

Reply:
I think it’s quite recent, because I have only found it now😅 I also haven’t used it before and haven’t tried it yet (just checked the replacement of intrinsics)

Reply:
Ah ok ! barely 3 weeks old! https://github.com/llvm/llvm-project/commit/93a9a8a8d90f5b9bb6965ebb1104082692d41833 !

I will update LLVM installation used in CI so that CI can pass.

Reply:
addressed